app/views/duties/_open_drop_modal.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="modal-content">                             
+  <div class="modal-header">
+    <h5 class="modal-title" id="exampleModalLabel">Drop</h5>
```

Comment:
Use `example-modal-label` for the ID.

And, probably better to rename that right